engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -20,6 +22,8 @@ class PeriodicProcessService(delegateDeploymentManager: DeploymentManager,
                              scheduledProcessesRepository: PeriodicProcessesRepository,
                              periodicProcessListener: PeriodicProcessListener,
                              additionalDeploymentDataProvider: AdditionalDeploymentDataProvider,
+                             deployMaxRetries: Int,
```

Comment:
Can we make a class like DeploymentRetryConfig? In case we want to add more stuff like backoff multiplier etc.

Reply:
:+1:

Reply:
done